pages/podcasts/index.vue

```diff
@@ -1,32 +1,25 @@
 <template>
-  <PodcastList :podcasts="podcasts" />
+  <PodcastList :podcasts="podcastsList" />
 </template>
 
 <script>
 import PodcastList from '@/components/podcasts/PodcastList'
+import { mapActions, mapGetters, mapState } from 'vuex'
 
 export default {
   components: { PodcastList },
   data() {
-    return {
-      podcasts: [
-        {
-          id: 1,
-          title: "Let's talk about Elixir!",
-          description: 'Lorem ipsum desu desu'
-        },
-        {
-          id: 2,
-          title: 'Management at Zenika Nantes',
-          description: 'Lorem ipsum desu desu'
-        },
-        {
-          id: 3,
-          title: 'Reactive programming in the back-end',
-          description: 'Lorem ipsum desu desu'
-        }
-      ]
-    }
+    return {}
+  },
+  computed: {
+    ...mapState('podcasts', ['podcastsList'])
+  },
+  async created() {
+    await this.initPodcastsList()
+  },
+  methods: {
+    ...mapActions('podcasts', ['initPodcastsList']),
+    ...mapGetters('podcasts', ['getPodcasts'])
```

Comment:
Getters won't work in methods object (You are not using it in this component, so i think you can remove it ^^)